src/miner_keys.erl

```diff
@@ -94,8 +111,19 @@ print_keys(_) ->
     MaybeB58 = fun(undefined) -> undefined;
                   (Key) -> libp2p_crypto:pubkey_to_b58(Key)
                end,
+    MaybeUUIDv4 = fun(undefined) -> undefined;
+                  (Key) ->
+                      %% only production blackspots read onboarding_key from a file
+                      OnboardingKey = get_onboarding_filename(),
+                      case filelib:is_file(OnboardingKey) of
+                          true ->
+                              string:trim(binary_to_list(Key));
+                          false ->
+                              libp2p_crypto:pubkey_to_b58(Key)
+                      end
+               end,
     Props = [{pubkey, MaybeB58(PubKey)},
-             {onboarding_key, MaybeB58(OnboardingKey)}
```

Comment:
Is Maybe B58 no longer used?

Reply:
> Is Maybe B58 no longer used?

It's used in the preceding line to validate `pubkey`. If I used `MaybeUUIDv4` instead then there would be no guarantee that `pubkey` is b58.

Reply:
whoops, you're right , missed that one